src/test/fixtures/config/forms/local/index.tsx

```diff
@@ -0,0 +1,4 @@
+import CoverLetterExtension from "./cover-letter-extension.json";
+import CoverLetterNoUiSchema from "./cover-letter-nouischema.json";
+export { CoverLetterExtension, CoverLetterNoUiSchema };
+export const documentFormsV2Local = [CoverLetterExtension, CoverLetterNoUiSchema];
```

Comment:
keep line 3 and let line 4 be handled at import file end? the same with other files

Reply:
done